beacon-chain/p2p/rpc_topic_mappings.go

```diff
@@ -116,7 +116,7 @@ var RPCTopicMappings = map[string]interface{}{
 	// DataColumnSidecarsByRange v1 Message
 	RPCDataColumnSidecarsByRangeTopicV1: new(pb.DataColumnSidecarsByRangeRequest),
 	// DataColumnSidecarsByRoot v1 Message
-	RPCDataColumnSidecarsByRootTopicV1: new(p2ptypes.BlobSidecarsByRootReq),
+	RPCDataColumnSidecarsByRootTopicV1: new(p2ptypes.DataColumnSidecarsByRootReq),
```

Comment:
Because now we don't use any more `blob` in place of `dataColumn`.
A solution could be:
- Remove the newsly created type
- Rename the old typ `BlobSidecarsByRootReq` into `sidecarsByRootReq`

What do you think?

Reply:
ah ok, I think its fine to leave it as you have done it. Our reusing of the blob types was more for code velocity rather than any correctness